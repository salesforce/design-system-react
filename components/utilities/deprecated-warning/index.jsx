/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */

// ### React
import React from 'react';

// ## Constants
import { DEPRECATED_WARNING } from '../../../utilities/constants';

import Alert from '../../alert';
import AlertContainer from '../../alert/container';

/**
 * A utility component that is used to highlight a deprecated component.
 */
const DeprecatedWarning = () => (
	<div className="slds-p-top_x-large">
		<AlertContainer>
			<Alert
				labels={{
					heading: (
						<span>
							<strong>Deprecated component.</strong> No fit and finish bug fixes
							will be accepted for this component.
						</span>
					),
				}}
				variant="error"
			/>
		</AlertContainer>
	</div>
);

// ### Display Name
DeprecatedWarning.displayName = DEPRECATED_WARNING;

export default DeprecatedWarning;
