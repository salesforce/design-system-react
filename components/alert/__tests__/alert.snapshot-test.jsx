/* eslint-env jest */
import {
	testDOMandHTML,
	testImageSnapshot
} from '../../../tests/snapshot-helpers';

import Info from '../__examples__/info';
import Warning from '../__examples__/warning';
import ErrorAlert from '../__examples__/error';
import Offline from '../__examples__/offline';
import Dismissable from '../__examples__/dismissable';
import CustomClassNames from '../__examples__/custom-class-name';

import { ALERT } from '../../../utilities/constants';

describe(ALERT, () => {
	test('Base DOM & HTML Snapshots look the same', () => {
		testDOMandHTML(Info);
		testDOMandHTML(Warning);
		testDOMandHTML(ErrorAlert);
		testDOMandHTML(Offline);
		testDOMandHTML(Dismissable);
		testDOMandHTML(CustomClassNames);
	});

	test('Info Image Snapshot looks the same', async () => {
		await testImageSnapshot(ALERT, 'Info');
	});

	test('Warning Image Snapshot looks the same', async () => {
		await testImageSnapshot(ALERT, 'Warning');
	});

	test('Error Image Snapshot looks the same', async () => {
		await testImageSnapshot(ALERT, 'Error');
	});

	test('Offline Image Snapshot looks the same', async () => {
		await testImageSnapshot(ALERT, 'Offline');
	});

	test('Dismissable Image Snapshot looks the same', async () => {
		await testImageSnapshot(ALERT, 'Dismissable');
	});

	test('Custom Class Name Image Snapshot looks the same', async () => {
		await testImageSnapshot(ALERT, 'Custom Class Name');
	});
});
