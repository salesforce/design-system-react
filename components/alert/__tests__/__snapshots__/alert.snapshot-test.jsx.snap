// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert Base DOM & HTML Snapshots look the same 1`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
  </div>
</div>
`;

exports[`Alert Base DOM & HTML Snapshots look the same 2`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2>
  </div>
</div>"
`;

exports[`Alert Base DOM & HTML Snapshots look the same 3`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_warning"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      warning
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#warning"
        />
      </svg>
      
    </span>
    <h2>
      Your browser is outdated. Your Salesforce experience may be degraded.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More Information
      </a>
    </h2>
  </div>
</div>
`;

exports[`Alert Base DOM & HTML Snapshots look the same 4`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_warning\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">warning</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#warning\\"></use></svg></span>
    <h2>Your browser is outdated. Your Salesforce experience may be degraded. <a href=\\"javascript:void(0);\\">More Information</a></h2>
  </div>
</div>"
`;

exports[`Alert Base DOM & HTML Snapshots look the same 5`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_error"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      error
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#ban"
        />
      </svg>
      
    </span>
    <h2>
      Your browser is currently not supported. Your Salesforce may be degraded.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More Information
      </a>
    </h2>
  </div>
</div>
`;

exports[`Alert Base DOM & HTML Snapshots look the same 6`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_error\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">error</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#ban\\"></use></svg></span>
    <h2>Your browser is currently not supported. Your Salesforce may be degraded. <a href=\\"javascript:void(0);\\">More Information</a></h2>
  </div>
</div>"
`;

exports[`Alert Base DOM & HTML Snapshots look the same 7`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_offline"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      offline
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#offline"
        />
      </svg>
      
    </span>
    <h2>
      You are in offline mode.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More information
      </a>
    </h2>
  </div>
</div>
`;

exports[`Alert Base DOM & HTML Snapshots look the same 8`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_offline\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">offline</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#offline\\"></use></svg></span>
    <h2>You are in offline mode. <a href=\\"javascript:void(0);\\">More information</a></h2>
  </div>
</div>"
`;

exports[`Alert Base DOM & HTML Snapshots look the same 9`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
    <button
      aria-controls={undefined}
      aria-describedby={undefined}
      aria-expanded={undefined}
      aria-haspopup={undefined}
      className="slds-button slds-button--icon-inverse slds-notify__close"
      disabled={false}
      id={undefined}
      onBlur={undefined}
      onClick={[Function]}
      onFocus={undefined}
      onKeyDown={undefined}
      onKeyPress={undefined}
      onKeyUp={undefined}
      onMouseDown={undefined}
      onMouseEnter={undefined}
      onMouseLeave={undefined}
      tabIndex={undefined}
      title="Close"
      type="button"
    >
      <svg
        aria-hidden="true"
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Close
      </span>
    </button>
  </div>
</div>
`;

exports[`Alert Base DOM & HTML Snapshots look the same 10`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2><button class=\\"slds-button slds-button--icon-inverse slds-notify__close\\" title=\\"Close\\" type=\\"button\\"><svg aria-hidden=\\"true\\" class=\\"slds-button__icon\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#close\\"></use></svg><span class=\\"slds-assistive-text\\">Close</span></button></div>
</div>"
`;

exports[`Alert Base DOM & HTML Snapshots look the same 11`] = `
<div
  className="slds-notify-container this-is-the-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info this-is-the-alert"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
  </div>
</div>
`;

exports[`Alert Base DOM & HTML Snapshots look the same 12`] = `
"<div class=\\"slds-notify-container this-is-the-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info this-is-the-alert\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2>
  </div>
</div>"
`;

exports[`Base DOM & HTML Snapshots 1`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
  </div>
</div>
`;

exports[`Base DOM & HTML Snapshots 2`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2>
  </div>
</div>"
`;

exports[`Base DOM & HTML Snapshots 3`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_warning"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      warning
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#warning"
        />
      </svg>
      
    </span>
    <h2>
      Your browser is outdated. Your Salesforce experience may be degraded.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More Information
      </a>
    </h2>
  </div>
</div>
`;

exports[`Base DOM & HTML Snapshots 4`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_warning\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">warning</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#warning\\"></use></svg></span>
    <h2>Your browser is outdated. Your Salesforce experience may be degraded. <a href=\\"javascript:void(0);\\">More Information</a></h2>
  </div>
</div>"
`;

exports[`Base DOM & HTML Snapshots 5`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_error"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      error
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#ban"
        />
      </svg>
      
    </span>
    <h2>
      Your browser is currently not supported. Your Salesforce may be degraded.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More Information
      </a>
    </h2>
  </div>
</div>
`;

exports[`Base DOM & HTML Snapshots 6`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_error\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">error</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#ban\\"></use></svg></span>
    <h2>Your browser is currently not supported. Your Salesforce may be degraded. <a href=\\"javascript:void(0);\\">More Information</a></h2>
  </div>
</div>"
`;

exports[`Base DOM & HTML Snapshots 7`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_offline"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      offline
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#offline"
        />
      </svg>
      
    </span>
    <h2>
      You are in offline mode.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More information
      </a>
    </h2>
  </div>
</div>
`;

exports[`Base DOM & HTML Snapshots 8`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_offline\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">offline</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#offline\\"></use></svg></span>
    <h2>You are in offline mode. <a href=\\"javascript:void(0);\\">More information</a></h2>
  </div>
</div>"
`;

exports[`Base DOM & HTML Snapshots 9`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
    <button
      aria-controls={undefined}
      aria-describedby={undefined}
      aria-expanded={undefined}
      aria-haspopup={undefined}
      className="slds-button slds-button--icon-inverse slds-notify__close"
      disabled={false}
      id={undefined}
      onBlur={undefined}
      onClick={[Function]}
      onFocus={undefined}
      onKeyDown={undefined}
      onKeyPress={undefined}
      onKeyUp={undefined}
      onMouseDown={undefined}
      onMouseEnter={undefined}
      onMouseLeave={undefined}
      tabIndex={undefined}
      title="Close"
      type="button"
    >
      <svg
        aria-hidden="true"
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Close
      </span>
    </button>
  </div>
</div>
`;

exports[`Base DOM & HTML Snapshots 10`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2><button class=\\"slds-button slds-button--icon-inverse slds-notify__close\\" title=\\"Close\\" type=\\"button\\"><svg aria-hidden=\\"true\\" class=\\"slds-button__icon\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#close\\"></use></svg><span class=\\"slds-assistive-text\\">Close</span></button></div>
</div>"
`;

exports[`Base DOM & HTML Snapshots 11`] = `
<div
  className="slds-notify-container this-is-the-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info this-is-the-alert"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
  </div>
</div>
`;

exports[`Base DOM & HTML Snapshots 12`] = `
"<div class=\\"slds-notify-container this-is-the-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info this-is-the-alert\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2>
  </div>
</div>"
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 1`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
  </div>
</div>
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 2`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2>
  </div>
</div>"
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 3`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_warning"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      warning
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#warning"
        />
      </svg>
      
    </span>
    <h2>
      Your browser is outdated. Your Salesforce experience may be degraded.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More Information
      </a>
    </h2>
  </div>
</div>
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 4`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_warning\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">warning</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#warning\\"></use></svg></span>
    <h2>Your browser is outdated. Your Salesforce experience may be degraded. <a href=\\"javascript:void(0);\\">More Information</a></h2>
  </div>
</div>"
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 5`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_error"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      error
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#ban"
        />
      </svg>
      
    </span>
    <h2>
      Your browser is currently not supported. Your Salesforce may be degraded.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More Information
      </a>
    </h2>
  </div>
</div>
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 6`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_error\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">error</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#ban\\"></use></svg></span>
    <h2>Your browser is currently not supported. Your Salesforce may be degraded. <a href=\\"javascript:void(0);\\">More Information</a></h2>
  </div>
</div>"
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 7`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_offline"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      offline
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#offline"
        />
      </svg>
      
    </span>
    <h2>
      You are in offline mode.
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        More information
      </a>
    </h2>
  </div>
</div>
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 8`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_offline\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">offline</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#offline\\"></use></svg></span>
    <h2>You are in offline mode. <a href=\\"javascript:void(0);\\">More information</a></h2>
  </div>
</div>"
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 9`] = `
<div
  className="slds-notify-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
    <button
      aria-controls={undefined}
      aria-describedby={undefined}
      aria-expanded={undefined}
      aria-haspopup={undefined}
      className="slds-button slds-button--icon-inverse slds-notify__close"
      disabled={false}
      id={undefined}
      onBlur={undefined}
      onClick={[Function]}
      onFocus={undefined}
      onKeyDown={undefined}
      onKeyPress={undefined}
      onKeyUp={undefined}
      onMouseDown={undefined}
      onMouseEnter={undefined}
      onMouseLeave={undefined}
      tabIndex={undefined}
      title="Close"
      type="button"
    >
      <svg
        aria-hidden="true"
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Close
      </span>
    </button>
  </div>
</div>
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 10`] = `
"<div class=\\"slds-notify-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2><button class=\\"slds-button slds-button--icon-inverse slds-notify__close\\" title=\\"Close\\" type=\\"button\\"><svg aria-hidden=\\"true\\" class=\\"slds-button__icon\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#close\\"></use></svg><span class=\\"slds-assistive-text\\">Close</span></button></div>
</div>"
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 11`] = `
<div
  className="slds-notify-container this-is-the-container"
>
  <div
    className="slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info this-is-the-alert"
    role="alert"
  >
    <span
      className="slds-assistive-text"
    >
      info
    </span>
    <span
      className="slds-m-right--x-small"
      title={undefined}
    >
      <svg
        aria-hidden="true"
        className="slds-icon slds-icon--x-small"
        style={undefined}
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#user"
        />
      </svg>
      
    </span>
    <h2>
      Logged in as John Smith (johnsmith@acme.com).
       
      <a
        href="javascript:void(0);"
        onClick={[Function]}
      >
        Log out
      </a>
    </h2>
  </div>
</div>
`;

exports[`SLDSAlert Base DOM & HTML Snapshots look the same 12`] = `
"<div class=\\"slds-notify-container this-is-the-container\\">
  <div class=\\"slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_info this-is-the-alert\\" role=\\"alert\\"><span class=\\"slds-assistive-text\\">info</span><span class=\\"slds-m-right--x-small\\"><svg aria-hidden=\\"true\\" class=\\"slds-icon slds-icon--x-small\\"><use xlink:href=\\"/assets/icons/utility-sprite/svg/symbols.svg#user\\"></use></svg></span>
    <h2>Logged in as John Smith (johnsmith@acme.com). <a href=\\"javascript:void(0);\\">Log out</a></h2>
  </div>
</div>"
`;
